Sources/StreamChat/Database/DTOs/ChannelDTO_Tests.swift

```diff
@@ -209,6 +211,72 @@ class ChannelDTO_Tests: XCTestCase {
         XCTAssertEqual(channel?.latestMessages.count, 10)
     }
 
+    func test_channelPayload_pinnedMessagesOlderThanOldestMessageAtAreIgnored() throws {
+        // Save a channel payload with 100 messages
+        let channelId: ChannelId = .unique
+        let oldPinnedMessage: MessagePayload<NoExtraData> = MessagePayload(
+            id: .unique,
+            type: .regular,
+            user: dummyUser,
+            createdAt: "2018-12-12T15:33:46.488935Z".toDate(),
```

Comment:
nit: Why not just use some more readable version of the date? The actual value doesn't matter, it just has to be older than a specific date.